Readme.md

```diff
@@ -127,62 +113,18 @@ final class ConfirmOrder
 }
 ```
 
-### CommandHandler
-
-```php
-/**
- * @Flow\Scope("singleton")
- */
-class ButtonCommandHandler
-{
-    /**
-     * @var ButtonRepository
-     * @Flow\Inject
-     */
-    protected $buttonRepository;
-
-    /**
-     * @param CreateButton $command
-     */
-    public function handleCreateButton(CreateButton $command)
-    {
-        $button = new Button($command->getIdentifier(), $command->getPublicIdentifier());
-        $button->changeLabel($command->getLabel());
-
-        $this->buttonRepository->save($button);
-    }
-}
-```
-
-### Monitoring
-
-You can disable (enabled by default) the command handler performance monitoring. The monitoring is implemented with AOP,
-check ```Settings.yaml``` for the configuration.
-
-## Domain
-
-* [x] **AggregateRoot**: implement your own based on ```AggregateRootInterface``` with ```AggregateRootTrait```
-* [x] **Repository**: implement your own based on ```RepositoryInterface```
-
-## Event
-
-### EventBus
-
-* [x] **EventBus**: default implementation, implement your own based on ```EventBusInterface```
-
-### EventListenerLocator
-
-* [x] **EventListenerLocator**: based on convention, implement your own based on ```EventListenerLocatorInterface```
+Notice that the commands do not have any dependency on the framework. They are pure POJOs and
+should optimally be immutable.
```

Comment:
This is a pretty deep topic and at least for validation outside the typical Flow validation system we do not have a consensus yet. I'll add a sentence anyway that gives a short introduction.